src/index.js

```diff
@@ -75,7 +68,10 @@ export default function profileStore(options = {}) {
 			performance.clearMarks(markName + START)
 			performance.clearMarks(markName + END)
 			performance.clearMeasures(getMarkLabel(markInfo, markType))
-			counter -= counterDelta
+			counter[markInfo.name]--
+			if (!counter[markInfo.name]) {
```

Comment:
Left a bug in my previous suggestion. This should fix it:
```suggestion
			if (counter[markInfo.name] <= 0) {
```